libraries/AP_RCTelemetry/AP_CRSF_Telem.cpp

```diff
@@ -282,7 +282,10 @@ void AP_CRSF_Telem::adjust_packet_weight(bool queue_empty)
      We start a "fast parameter window" that we close after 5sec
     */
     bool expired = (now_ms - _custom_telem.params_mode_start_ms) > 5000;
-    if (!_custom_telem.params_mode_active && _pending_request.frame_type > 0) {
+    if (!_custom_telem.params_mode_active
+        && _pending_request.frame_type > 0
+        && _pending_request.frame_type != AP_RCProtocol_CRSF::CRSF_FRAMETYPE_PARAM_DEVICE_INFO
```

Comment:
I made a typo in the code I presented here: https://github.com/ArduPilot/ardupilot/issues/18623#issuecomment-919088318

This should be CRSF_FRAMETYPE_PARAM_DEVICE_PING instead of CRSF_FRAMETYPE_PARAM_DEVICE_INFO.  This probably worked since the code also had a quick exit.

Reply:
This should be INFO - the *pending* request is INFO which is in response to a PING

Reply:
yes. right you are